Tests/AppTests/RSSTests.swift

```diff
@@ -40,20 +44,21 @@ class RSSTests: SnapshotTestCase {
         let feed = RSSFeed(title: "feed title", description: "feed description",
                            link: "https://SwiftPackageIndex.com",
                            items: [
-                            RecentPackage(id: UUID(),
-                                          repositoryOwner: "owner0",
-                                          repositoryName: "repo0",
-                                          packageName: "package0",
-                                          packageSummary: "summary0",
-                                          createdAt: Date(timeIntervalSince1970: 0)).rssItem,
-                            RecentPackage(id: UUID(),
-                                          repositoryOwner: "owner1",
-                                          repositoryName: "repo1",
-                                          packageName: "package1",
-                                          packageSummary: "summary1",
-                                          createdAt: Date(timeIntervalSince1970: 1)).rssItem]
+                            RecentPackage.mock(id: .id0,
+                                               repositoryOwner: "repositoryOwner0",
+                                               repositoryName: "repositoryName0",
+                                               packageName: "packageName0",
+                                               packageSummary: "packageSummary0",
+                                               createdAt: .t0).rssItem,
+                            RecentPackage.mock(id: .id1,
+                                               repositoryOwner: "repositoryOwner1",
+                                               repositoryName: "repositoryName1",
+                                               packageName: "packageName1",
+                                               packageSummary: "packageSummary1",
+                                               createdAt: .t1).rssItem
+                           ]
         )
```

Comment:
I'm not sure there's any huge value in using the mock where every field is being specified, but it brings us up to date with using `.t0` and `.id0` helpers.